nextcord/client.py

```diff
@@ -2023,33 +2039,171 @@ def get_application_command(self, command_id: int) -> Optional[BaseApplicationCo
         """
         return self._connection.get_application_command(command_id)
 
-    def get_application_command_from_signature(
-        self, name: str, cmd_type: Union[int, ApplicationCommandType], guild_id: Optional[int]
-    ) -> Optional[BaseApplicationCommand]:
+    def __actual_get_application_command_from_signature(
+        self,
+        *,
+        type: Union[int, ApplicationCommandType],
+        qualified_name: str,
+        guild_id: Optional[int],
+        search_locales: bool = False,
+    ) -> Optional[Union[BaseApplicationCommand, SlashApplicationSubcommand]]:
         """Gets a locally stored application command object that matches the given signature.
 
+        .. versionadded:: 2.0
+
+        .. versionchanged:: 2.3
+            it's no longer possible to pass ``None`` to the ``name`` parameter.
```

Comment:
Shouldn't this part be merged in with the part below? Because the `name` parameter was renamed to `qualified_name`

Reply:
Not really related imo